hw06_pipeline_execution/go.mod

```
@@ -1,10 +1,9 @@
-module github.com/fixme_my_friend/hw06_pipeline_execution
+module github.com/director74/homeworks/tree/master/hw06_pipeline_execution
```

```suggestion
module github.com/director74/homeworks/hw06_pipeline_execution
```
`tree/master` - роутинг UI GitHub, и не имеет отношения к Go модулям

---

спасибо, скорректировал